src/components/ItaliaTheme/View/Commons/RelatedArticles.jsx

```diff
@@ -21,10 +21,11 @@ const RelatedArticles = ({
     <article
       id={id}
       className={cx('it-page-section', 'anchor-offset', { 'mt-5': !noMargin })}
+      aria-labelledby={`header-${id}`}
     >
       {title && show_title ? (
         title_size === 'h5' ? (
-          <h5>{title}</h5>
+          <h5 id={`header-${id}`}>{title}</h5>
```

Comment:
Qui l'id non c'era proprio per fare in modo che non comparisse nel menù laterale, perché si mette h5 quando è un sottotitolo meno importante. Se non ci hai visto altri problemi, lo lascerei senza.